forge-gui-mobile/src/forge/adventure/scene/NewGameScene.java

```diff
@@ -446,10 +533,31 @@ private void showModeHelp() {
                 }
                 break;
             case Chaos:
-                summaryText.append("Mode: Chaos\n\nYou (and all enemies) will receive a random preconstructed deck.\n\nWarning: This will make encounter difficulty vary wildly from the developers' intent");
+                summaryText.append("""
+                        Mode: Chaos
+                        
```

Comment:
does hard converting these linebreaks work for all platforms?

Reply:
AFAIK, they are part of the Java language specification since Java 15 and thus not platform-dependent.
But this is an "unnecessary" alteration regarding the objective of this issue, so I am happy to revert these and align the added text.

Reply:
I've reverted to the way it was before, just in case.